src/dramatiq_abort/backends/redis.py

```diff
@@ -46,3 +46,10 @@ def notify(self, key: bytes, ttl: int) -> None:
             pipe.rpush(key, b"x")
             pipe.pexpire(key, ttl)
             pipe.execute()
+
+    def notify_many(self, keys: List[bytes], ttl: int) -> None:
```

Comment:
You could also rewrite `notify` to simply call `self.notify_many([key], ttl)`. This would avoid having duplicate implementation.